src/WPBT/WPBT_Core.php

```diff
@@ -267,7 +267,7 @@ public function get_next_version( $preferred_version ) {
 			$next_version = __( 'next development version', 'wordpress-beta-tester' );
 		} else {
 			// show all versions that may come next.
-			$next_version = implode( ' or ', $next_version ) . ', ' . __( 'whichever is released first', 'wordpress-beta-tester' );
+			$next_version = wp_sprintf( __( 'version %l', 'wordpress-beta-tester' ), $next_version ) . ', ' . __( 'whichever is released first', 'wordpress-beta-tester' );
```

Comment:
I had no idea about this function, `wp_sprintf()`. Interesting.

Reply:
It is indeed. I'm trying to change it from "and" to "or" :)

Reply:
We need to check the requirements of this function. We are limited in that we must currently support WP 3.1 and PHP 5.2. I think the use of `...$args` in the function might be an issue. Need to investigate.

Reply:
It still might be ok. Just need to make sure.

Reply:
Looks like the filter `wp_sprintf_l` is where the `and` comes from.